x/manifest/types/codec.go

```diff
@@ -3,19 +3,10 @@ package types
 import (
 	"github.com/cosmos/cosmos-sdk/codec"
 	"github.com/cosmos/cosmos-sdk/codec/types"
-	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/msgservice"
 )
 
-var amino = codec.NewLegacyAmino()
-
-func init() {
```

Comment:
I remember removing this in the past for a network & it causing issues. They did come up from SDK v47 -> v50 though, so it's possible it was specific to that vs being on SDK v50 from the start like manifest. Just want to call it out now

Reply:
Do you remember the issue?